coordinator/src/node/storage.rs

```diff
@@ -161,6 +164,11 @@ impl node::Storage for NodeStorage {
         Ok(channel)
     }
 
+    fn get_channel_by_payment_hash(&self, _payment_hash: String) -> Result<Option<Channel>> {
+        // the payment hash is not stored on the coordinator side.
+        Ok(None)
```

Comment:
🔧 `unimplemented!` is saner if this is never called. Or are we actually calling it? 😆

Reply:
it shouldn't be called :)